.buildkite/pipelines/emergency_release.yml

```diff
@@ -1,3 +1,18 @@
+env:
+  VERSION: ${BUILDKITE_COMMIT:0:12}
+  DOCKER_IMAGE_TAG: git-${VERSION}
+  DOCKER_IMAGE: docker.elastic.co/kibana-ci/kibana-serverless
+  EMERGENCY_RELEASE_REMOTE_SERVICE_CONFIG: https://raw.githubusercontent.com/elastic/serverless-gitops/main/gen/gpctl/kibana/emergency.yaml
+
 steps:
-  - label: Placeholder
-    command: echo "Hello!"
+  - label: ':label: Create tag'
+    command: .buildkite/scripts/serverless/create_deploy_tag/create_fix_tag.sh
+    if: build.branch =~ /^deploy-fix@.*\$/
+
+  - label: ':argo: Emergency Release'
+    if: build.tag =~ /^deploy-fix@-.*\$/ && build.branch =~ /^deploy-fix@.*\$/
```

Comment:
I need to test this still but want to sync on a mock promotion before sending out notifications.

Reply:
Do you have the extra `-` in the tag regex to ensure that a release isn't triggered, or is that a typo?

How do you plan to test this? Alternatively I think it's fair to do a full test of this pipeline and actually perform an emergency release once it's merged

Reply:
I pushed https://github.com/elastic/kibana/pull/176258/commits/5d36c21f7cafa489019c0537f1629d3607561195

Reply:
> How do you plan to test this? Alternatively I think it's fair to do a full test of this pipeline and actually perform an emergency release once it's merged

:+1: .  I can hack the regex's here to let this branch be tested, but merging as is will give us a more exact test.